trade-finance-manager-ui/server/errors/invalid-update-report-status-request.error.test.ts

```diff
@@ -0,0 +1,22 @@
+import { InvalidUpdateReportStatusRequestError } from './invalid-update-report-status-request.error';
+
+describe('InvalidUpdateReportStatusRequestError', () => {
+  const message = 'an error message';
+
+  it('exposes the message it was created with', () => {
+    // Act
+    const exception = new InvalidUpdateReportStatusRequestError(message);
+
+    // Assert
+    expect(exception.message).toBe(message);
+  });
+
+  // Test will pass if we ever update to es6
```

Comment:
I saw in the tsconfig we've now downgraded to es5 from es6 due to some other issues. It would to resolve those issues so we can keep es6 if possible.

Reply:
Reverted to `es6`